libs/metrics/icu_capacity.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+import pandas as pd
+
+from covidactnow.datapublic.common_fields import CommonFields
+
+from libs.datasets import timeseries
+
+
+def calculate_icu_capacity(region_df: pd.DataFrame):
+    """Calculate ICU Capacity ratio.
+
+    Args:
+        region_df: Data for a specific region.
+
+    Returns: np.nan if data missing or pd.Series of ICU capacity.
+    """
+    # TODO(chris): Why does the common fields break if it's not in the array?
```

Comment:
Can you resolve this TODO or else expand on it?  I can't quite tell what it's talking about or what action we'd want to take.